source/_integrations/nordpool.markdown

```diff
@@ -137,7 +137,9 @@ Currency:
 
 The public API only allows us to see past pricing information for up to 2 months.
 
-Tomorrow's prices are typically released around 13:00 CET, and trying to get them before that time will generate an error that needs to be considered in such a case.
+Although Nord Pool operates in the CET/CEST timezone, all data is returned in UTC. Depending on how the data is consumed or manipulated, you may need to consider this.
+
+Tomorrow's prices are typically released around 13:00 CET/CEST, and trying to get them before that time will generate an error that needs to be considered in such a case.
```

Comment:
what do you mean by "error that needs to be considered in such a case". what do I do with it?

Reply:
That text was already there but if a user ask for tomorrows prices "too soon" it will result in `ServiceValidationError` and not just an empty list.
We should change that so it doesn't raises an error but that's how it's done at the moment.

Reply:
ah, ok. thx.